docs/content/mobile/ios/setup.md

```diff
@@ -2,13 +2,18 @@
 
 ## Get the dependencies for developing iOS games
 
+> **Note:** iOS development is only supported on macOS.
+
   1. Follow the steps in [Get the dependencies]({{urlRoot}}/setup-and-installing) and additionally install **iOS build support** for Unity.
   1. Install iOS prerequisites
     * [Xcode](https://developer.apple.com/xcode/)
       * Once installed, open Xcode. It might prompt you to accept user agreements.
     * (Optional) [Unity Remote](https://itunes.apple.com/gb/app/unity-remote-5/id871767552?mt=8) - this is Unity’s solution for faster development iteration times.
 
 ## Set up your Unity Editor
-After installing these dependencies, open the Unity project in `<path-to-your-project>/workers/unity`.
 
-  1. In the Unity Editor, go to **File** > **Build Settings**. Select **iOS** and then click on **Switch Platform**.
+Most of your interactions with the GDK will happen inside the Unity Editor. To get started:
```

Comment:
```suggestion
Most of your interactions with the GDK happen inside the Unity Editor. To get started:
```

active voice as Laura would say ;)

Reply:
OMG Jess. You have absorbed me.

Reply:
thanks for using the Suggested Changes flow here